src/content/chainlink-local/build/ccip/foundry/local-simulator.mdx

```diff
@@ -0,0 +1,253 @@
+---
+section: chainlinkLocal
+date: Last Modified
+title: "Using CCIP local simulator in your Foundry project"
+isIndex: false
+---
+
+import Common from "@features/chainlink-local/Common.astro"
+
+<Common callout="importPackage" />
+
+You can use Chainlink Local to run CCIP in a localhost environment within your Foundry project. To get started quickly, we will use the [CCIP Starter Kit - Foundry](https://github.com/smartcontractkit/ccip-starter-kit-foundry). This project is a Foundry boilerplate that includes the Chainlink Local package and several CCIP examples.
+
+## Prerequisites
+
+1. In a terminal, clone the [CCIP Starter Kit - Foundry](https://github.com/smartcontractkit/ccip-starter-kit-foundry) repository and change directories.
+
+   ```shell
+   git clone https://github.com/smartcontractkit/ccip-starter-kit-foundry.git && \
+   cd ./ccip-starter-kit-foundry/
+   ```
+
+1. Run `npm install` to install the dependencies. This will install the [Chainlink Local](https://github.com/smartcontractkit/chainlink-local) package and other required packages.
```

Comment:
Remove "Run [x command]" from the text description because it's followed by the same command